src/helpers/judgeZero.ts

```diff
@@ -0,0 +1,119 @@
+import axios from 'axios';
+import Eris, { Embed, EmbedOptions, MessageContent } from 'eris';
+
+const client = axios.create({
+    baseURL: 'https://judge0-ce.p.rapidapi.com',
+    headers: {
+        'content-type': 'application/json',
+        'x-rapidapi-key': process.env.JUDGE0_API_KEY,
+        'x-rapidapi-host': 'judge0-ce.p.rapidapi.com',
+        useQueryString: true,
+    },
+});
+
+interface JudgeZeroLanguage{
+    id : number,
+    name : string
+}
+
+interface JudgeZeroRequestBody{
+    language_id : number
+    source_code : string
+}
+
+interface JudgeZeroPostResponse{
+    token : string
+}
+
+interface JudgeZeroGetResponse{
+    stdout : string
+    stderr : string
+    message : string
+    memory : number
+    time : string
+    compile_output : string
+}
+
+const judgeZeroSubmissionParams = {
+    base64_encoded: true,
+    fields: '*',
+};
+
+export async function getAvailableLanguages() : Promise<JudgeZeroLanguage[]> {
+    const res = await client.get('/languages');
+    return res.data;
+}
+
+export async function createEvalRequest(body : JudgeZeroRequestBody) : Promise<JudgeZeroPostResponse> {
+    body.source_code = Buffer.from(body.source_code, 'binary').toString('base64');
+
+    const res = await client.post('/submissions', body, {
+        params: judgeZeroSubmissionParams,
+    });
+
+    return res.data;
+}
+
+export async function getEvalResult(token : string) : Promise<JudgeZeroGetResponse> {
+    const res = await client.get(`/submissions/${token}`, {
+        params: judgeZeroSubmissionParams,
+    });
```

Comment:
This would work when the code immediately evaluates and should be able to handle most situations, however, you might want to have a look at: https://ce.judge0.com/#statuses-and-languages-status
If the status code is 2, the GET request will still return, but the code will still be being evaluated. That's why Hero uses a loop for it.

Other than that, I think it's good to go 🙆🏻‍♂️